### ISSUE TITLE
AUR entry

### ISSUE THREAD

**Issue by Noammac** [illinoisjackson/even-better-ls#28]
I believe we could get more exposure for this project (therefore, more maintainers and bug-fixers) if we publish it to the AUR. I volunteer to try and make an entry and update the project but I'm still quite new at this and would like to consult some guides first.
If anyone else (preferably with more experience with pushing things to the AUR) would like to volunteer and replace me I would be very glad.

**Comment by e-n-l** [comment]
Any update on this?

**Comment by Noammac** [comment]
As far as we know Illinois might be working on something, though we're not
sure to what extent. The package itself is outdated and there's no one
who's working on this.

**Comment by DavidBittner** [comment]
Hey guys, I wrote a PKGBUILD for this. Curious how @illinoisjackson would like to me to contribute it. Should I just push it straight to the AUR?

**Comment by ruro** [comment]
@DavidBittner does your PKGBUILD work? I thought there were still some issues with coreutils, that broke the build. At least the `install.sh` script didn't work last time I checked.

Also, I am curious, how are you handling the `append_to_profile.sh` business.

**Comment by DavidBittner** [comment]
You're right, the install script doesn't work. Somebody updated the tar to 8.2.9 which does not work with the build, you must edit it to use 8.2 instead.

It successfully compiles and installs, but I can't seem to get the icons working. I never actually have used this before, so I wrote the PKGBUILD for my own sake of installation. Not sure if I'm doing something wrong.

In regard to the append_to_profile biz, I was going to go ahead and echo it at the end of installation so that the user can decide how they want to handle it in a `.install` file.
